scss/styles.scss

```diff
@@ -95,11 +97,49 @@
 .nav-title {
   align-self: center;
   font-size: $font-big;
-  padding-right: 25px;
 }
 
 .nav-logo {
   align-self: center;
-  padding-left: 25px;
   width: 200px;
 }
+
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  justify-content: center;
+  left: 0;
+  padding: 50px;
+  position: absolute;
+  width: 100%;
+}
+
+.footer-title {
+  color: $white;
+  font-size: $font-big;
+  margin-bottom: 25px;
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  width: 50%;
```

Comment:
this is too big in some screens, I'd recommend you change it to:
```
width: 100%;
max-width: 300px;
```

that way It will look good in big and small screens